src/gui/qgsbrowserdockwidget.h

```diff
@@ -126,43 +126,100 @@ class GUI_EXPORT QgsBrowserDockWidget : public QgsDockWidget, private Ui::QgsBro
      */
     Q_DECL_DEPRECATED void removeFavorite() SIP_DEPRECATED;
 
-    //! Refresh browser view model (and view)
+    /**
+     * Refresh the browser model and view.
+    */
     void refresh();
```

Comment:
Did you mean to deprecate this one too? I see you've avoided using it in other parts of the code (e.g. https://github.com/qgis/QGIS/pull/44363/files#diff-e554c4f3bac787da415f18a1e53f16209b0c5fa63702a69455d805d92ca6e891R119)

Reply:
I figure that one is pretty safe/conveniant to leave